odpf/compass/v1beta1/service.proto

```diff
@@ -1036,6 +1051,32 @@ message GetAssetByVersionResponse {
   Asset data = 1;
 }
 
+message CreateAssetProbeRequest {
+  message BaseProbe {
```

Comment:
@StewartJingga any reason to calli it `BaseProbe` ?

Reply:
@ravisuhag  No major reason, mostly because I was following how we are doing it for other request payload as seen [here](https://github.com/odpf/proton/blob/main/odpf/compass/v1beta1/service.proto#L929).

Also the scope of this message only for `CreateAssetProbeRequest`, so if it needs to be changed I don't see any issue with it.

Reply:
I think in both the cases we should keep it without Base?  wdyt?

Reply:
Yes. lets do that.

Reply:
@ravisuhag updated `BaseProbe` to `Probe`

And here is the PR for `BaseAsset` to `Asset`
https://github.com/odpf/proton/pull/188